.golangci.yml

```diff
@@ -10,6 +10,14 @@ linters-settings:
   golint:
     min-confidence: 0
 
+issues:
+  include:
```

Comment:
Is it possible to always include all?

Reply:
I think it still makes sense for some low risk issues to be excluded. Is it possible to get the security team to audit low risk issues instead of add `gosec` tag?